superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -40,6 +46,29 @@ import { exploreChart } from '../../explore/exploreUtils';
 import { CtasEnum } from '../actions/sqlLab';
 import { Query } from '../types';
 
+const updateDatset = async (
+  datasetId: number,
+  dbId: number,
+  sql: string,
+  columns: Array<Record<string, any>>,
+  overrideColumns: boolean,
+) => {
+  const endpoint = `api/v1/dataset/${datasetId}?override_columns=${overrideColumns}`;
+  const headers = { 'Content-Type': 'application/json' };
```

Comment:
Not 100% sure, but I think this should be the default. Do you need it?

Reply:
Yea i get 400 back without setting the `Content-Type` for the header

Reply:
cool. It's probably just superagent that has that default then.